helpers/concat.js

```diff
@@ -6,8 +6,13 @@
  *  {{@lang (concat 'products.reviews.rating.' this) }}
  */
 function helper(paper) {
-    paper.handlebars.registerHelper('concat', function (value, otherValue) {
-        return new paper.handlebars.SafeString(value + otherValue);
+    paper.handlebars.registerHelper('concat', function (value, otherValue, returnSafeString =  true) {
```

Comment:
I would rename this parameter to `returnUnsafe` and default it to false, which will preserve existing behavior.